### DISCUSSION TITLE
GDS frontend V4.2.0 upgrade

### DISCUSSION THREAD

**Discussion by MarkRobinsonDfe** [x-govuk/govuk-frontend-aspnetcore#233] [Q&A] [answered]
hi @gunndabad ,

Do you plan to upgrade this nuGet to support GDS frontend V4.2.0?

As the following classes are not in the NuGet:-
https://govuk-prototype-kit.herokuapp.com/docs/templates/task-list

Obviously, with some guidance, I could probably add in the tag helpers for this

Cheers

**Comment by gunndabad** [comment, marked as answer]
Hi

Yes, I'm working on an upgrade to the latest GDS version. That said, I don't believe any CSS classes from the task list pattern from the prototype kit would be available even with the upgrade. 

--
James

**Reply by MarkRobinsonDfe** [reply]
thanks James
